shared/rspec/support/capybara.rb

```diff
@@ -1,27 +1,15 @@
-require 'capybara'
-require 'capybara/rspec'
-require 'capybara/poltergeist'
-
-Capybara.default_host = 'http://0.0.0.0:3000'
-Capybara.javascript_driver = :poltergeist
-Capybara.default_max_wait_time = 15
-Capybara.wait_on_first_by_default = 15
-
-Capybara.register_driver :poltergeist do |app|
-  options = {
-    js_errors: false,
-    phantomjs_options: [
-      '--ssl-protocol=any'
-    ],
-    window_size: [2048, 2048]
-  }
-  Capybara::Poltergeist::Driver.new(app, options)
+# https://github.com/rails/rails/pull/30876
```

Comment:
@anduonghien i like the configuration from [thoughtbot](https://robots.thoughtbot.com/headless-feature-specs-with-chrome)

Major advantage: Ease of switching between headless and browser

```
require "selenium/webdriver"

Capybara.register_driver :chrome do |app|
  Capybara::Selenium::Driver.new(app, browser: :chrome)
end

Capybara.register_driver :headless_chrome do |app|
  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
    chromeOptions: { args: %w(headless disable-gpu) }
  )

  Capybara::Selenium::Driver.new app,
    browser: :chrome,
    desired_capabilities: capabilities
end

Capybara.javascript_driver = :headless_chrome
```

Reply:
@abhinavmsra Acutally this configuartion is old and it only for Capybara, the Rails team had implement everything, if we go with config like that, I think it doesn't make sence because we don't use any config from System Test.

How do you think?

Reply:
@anduonghien @olivierobert it seems its not required any longer.